# 0x05. Python - Exceptions

An exception is an event, which occurs during the execution of a program that disrupts the normal flow of the program's instructions. In general, when a Python script encounters a situation that it cannot cope with, it raises an exception. An exception is a Python object that represents an error.

## Table of contents
Files | Description
----- | -----------
[0-safe_print_list.py](./0-safe_print_list.py) | Python function that prints x elements of a list
[1-safe_print_integer.py](./1-safe_print_integer.py) | Python function that prints an integer with "{:d}".format()
[2-safe_print_list_integers.py](./2-safe_print_list_integers.py) | Python function that prints the first x elements of a list and only integers
[3-safe_print_division.py](./3-safe_print_division.py) | Python function that divides 2 integers and prints the result
[4-list_division.py](./4-list_division.py) | Python function that divides element by element 2 lists
[5-raise_exception.py](./5-raise_exception.py) | Python function that raises a type exception
[6-raise_exception_msg.py](./6-raise_exception_msg.py) | Python function that raises a name exception with a message

## Information:

[1](https://www.tutorialspoint.com/python/python_exceptions.htm)

[Diego Fernando Jojoa Yand√∫n](https://github.com/diegojojoayandun)

